index.html

```diff
@@ -83,62 +89,83 @@ <h2>Main Program</h2>
-            <p>
-              Listen to the speakers from various countries about the messages
-              of sharing and opening.
-            </p>
-          </div>
-          <!-- card 5 -->
+        <div class="program">
+          <div class="program dflex">
+            <div class="card-content dflex">
+              <img src="./images/conference-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Lecture</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 2 -->
+            <div class="card-content dflex">
+              <img src="./images/schedule-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Projects</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 3 -->
+            <div class="card-content dflex">
+              <img src="./images/chat-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Forum</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 4 -->
+            <div class="card-content dflex">
+              <img src="./images/main-event-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Workshop</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+            <!-- card 5 -->
 
-          <div class="card-content dflex">
-            <img src="./images/ignite-icon.png" alt="conference-icon" />
-            <h3 class="prg-title prg-1">Network</h3>
-            <p>
-              Listen to the speakers from various countries about the messages
-              of sharing and opening.
-            </p>
-          </div>
-          <div class="desktop-only">
-            <a href="#">SEE THE WHOLE PROGRAM</a>
+            <div class="card-content dflex">
+              <img src="./images/ignite-icon.png" alt="conference-icon" />
+              <h3 class="prg-title prg-1">Network</h3>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
           </div>
         </div>
-        <div class="join">Join global DevCon 2022</div>
+        <div class="desktop-only see-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="join mobile-only">Join global DevCon 2022</div>
       </div>
       <div class="speakers-section"></div>
```

Comment:
I think that it'll be more semantic to have the speakers section to be within a `section` tag. What do you think?

Reply:
Hello, @emmiiorji great work there! I will change one of the footers to a div. On the speaker's section, yeah, it will be better. However, all the changes you requested are not part of the core requirements of this project, please help me understand why they were not categorized under [optional] requirements. I am asking this to fully understand the review process.

Reply:
I have changed them

Reply:
Hi @jaredatandi,

Thanks for your response. The changes requested are for the following reasons:

- As mentioned [here](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation), the project required that you record a video where you present the project.
- As stated in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#htmlcss--javascript-requirements), you were asked to follow HTML and CSS best practices.

I hope the explanation helps.

Reply:
ooh! I now understand

Reply:
You forgot to request for the video, I hope the next reviewer is not here before I record it.

Reply:
I did @jaredatandi.. It's on the summary:

![image](https://user-images.githubusercontent.com/20554583/203711125-ae141806-41d5-46fe-96e8-05b514944c4e.png)